internal/cmd/local/command.go

```diff
@@ -9,14 +9,17 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/local"
 )
 
-type Command struct {
+const imageName = "523370736235.dkr.ecr.us-west-2.amazonaws.com/confluentinc/kafka-local:latest"
+const testTopicName = "jsontest"
+
+type localCommand struct {
```

Comment:
```suggestion
type command struct {
```

Reply:
Do we still need this struct? This command isn't authenticated, and I don't think we're using `ch` or `cc` anymore.

Reply:
`cannot define new methods on non-local type cmd.CLICommand` so we still need a struct

Reply:
i changed it to `command`